services/TodoService.js

```diff
@@ -0,0 +1,95 @@
+const dayjs = require('dayjs');
+const Todo = require('../schemas/todo');
+
+class TodoService {
+  /**
+   *
+   * @type {Todo[]}
+   */
+  tasks = new Array(10).fill(new Todo({ title: 'test title' }));
+
+  /**
+   * Get active tasks
+   * @returns {Todo[]}
+   */
+  getActive() {
+    return this.tasks.filter(task => !task.isCompleted);
+  }
+
+  /**
+   * Get all tasks
+   * @returns {Todo[]}
+   */
+  getAll() {
+    return this.tasks;
+  }
+
+  /**
+   * Complete single task
+   * @param taskId {string}
+   */
+  complete(taskId) {
+    for (let task of this.tasks) {
```

Comment:
use `map` instead of `for of` to create new correct array for future file db integration.